Tests/CADCacheAdvanceTests/CADCacheAdvanceTests.m

```diff
@@ -16,14 +16,13 @@
 //
 
 #if GENERATED_XCODE_PROJECT
-// These imports work when working within the generated CacheAdvance.xcodeproj used in CI.
+// This import works when working within the generated CacheAdvance.xcodeproj used in CI.
 #import <CADCacheAdvance/CADCacheAdvance-Swift.h>
-#import <XCTest/XCTest.h>
 #else
-// These imports work when working within Package.swift.
+// This import works when working within Package.swift.
 @import CADCacheAdvance;
-@import XCTest;
 #endif
+@import XCTest;
```

Comment:
I do like the idea of symmetry but I think this also makes a bit more clear why we are diverging in that one case. Before I wasn't immediately sure if it had to do with `#import` vs. `@import`.

Reply:
Makes a lot of sense. Reducing the need for a close reading is a good strategy for long-term maintenance (and for enabling others to contribute!).